akka-stream-tests/src/test/scala/akka/stream/actor/ActorPublisherSpec.scala

```diff
@@ -139,7 +139,7 @@ class ActorPublisherSpec extends AkkaSpec(ActorPublisherSpec.config) with Implic
       val probe = TestProbe()
       val ref = system.actorOf(testPublisherProps(probe.ref))
       val p = ActorPublisher[String](ref)
-      val s = StreamTestKit.SubscriberProbe[String]()
+      val s = TestSubscriber.probe[String]()
       p.subscribe(s)
       val sub = s.expectSubscription
```

Comment:
I always disliked this style. It is the subscriber that requests, so it reads weird to call the method on the subscription. Maybe we should add an Auto style subscriber probe that handles the subscription part automatically.